<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="BtnAdd">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>增加</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="BtnDel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="FindSort">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>全部</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>真实姓名</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>学号/教工号</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>用户名</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>用户权限</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>密码</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="BtnFind">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>查找</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="BtnSure">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="BtnCancel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
